### DISCUSSION TITLE
Bring back currently played map to online players dashboard

### DISCUSSION THREAD

**Discussion by yeahirai** [ppy/osu#30987] [Ideas] [answered]
Being able to see what friends and other people are playing has always been a pretty neat feature of the online players dashboard. Not only it served as a social aspect as being able to peek at the kind of map that are being played, but it also had the function of informing what is being played so that you can make a decision of spectating or not.
![osu!_rFueZErdR6](https://github.com/user-attachments/assets/3fca4ed7-79c5-4b1d-8f0c-5087e61b55f3)
![osu!_RwVinO6xwG](https://github.com/user-attachments/assets/fc42188f-26af-407d-b8b7-1139c46a0cf8)
I assume that this feature being replaced with a message like "clicking circles" is just a placeholder while this area is not fully developed yet, so i just want to leave this here to remind one of the contributors of how much this feature is appreciated.

**Comment by SupDos** [comment, marked as answer]
Already being considered in the new user card design https://github.com/ppy/osu-design/discussions/1 (Extension Area)

**Reply by yeahirai** [reply]
Nice
